spec/catchers/googlebooks_spec.rb

```diff
@@ -1,10 +1,10 @@
 require 'rails_helper'
 
-describe 'GoogleBooks' do
+describe GoogleBooks do
   context 'catch a book by' do
     it 'isbn' do
       VCR.use_cassette('search-isbn') do
-        book = GoogleBooks.search('isbn:9781118084786').first
+        book = described_class.search('isbn:9781118084786').first
```

Comment:
ok, but why change a code only to be cute?

In other words: this code is not related with your PR :| 

* Your PR: "Create book by isbn";
* This file changes: "be cute".

Is it related?